templates/index.html

```diff
@@ -7,10 +7,19 @@
   <h1 class="u-off-screen">Snapcraft</h1>
   <div class="row">
     {% if nps %}
-      <div class="p-notification p-snapcraft-nps-banner">
-        <p class="p-notification__response">
-          Thank you so much 🙏. Your feedback is really valuable to us.
-        </p>
+      <div class="p-notification p-snapcraft-nps-banner js-notification-holder">
+        <div class="p-notification__response">
+          <p class="u-no-margin--bottom u-no-padding--top">
+            Thank you so much 🙏. Your feedback is really valuable to us. <a href="#" class="js-nps-comment-toggle">Provide further comments</a>
+          </p>
+          <div class="p-snapcraft-nps-banner__form js-nps-comment u-hide">
+            <form class="--dark">
+              <textarea rows="5" name="snapcraftNPSNotes"></textarea>
+              <button type="submit" class="has-icon u-float--right">Send</button>
+            </form>
+          </div>
+        </div>
+        <a href="#" class="p-notification__action"><i class="p-icon--close"></i></a>
```

Comment:
Why not using vanilla built in notification support for `<button class="p-icon--close">` without custom "__action"?

```html
<div class="p-notification" id="notification">
  <p class="p-notification__response">
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt ipsum nemo autem reiciendis nulla tempore natus repudiandae dolorem. Corporis maxime, iure maiores repellat, odit facilis!
  </p>
  <button class="p-icon--close" aria-label="Close notification" onclick="closeNotification('notification')">Close</button>
</div>
```

Reply:
A thanks, will update this - I couldn't find it in the docs so thought I'd imagined it :+1:!